run.sh

```diff
@@ -69,13 +69,8 @@ case "$deployment_action" in
         deploy_ses
         ;;
     "deploy_caasp")
-        # this is CaaSP 3
         deploy_caasp
         ;;
-    "deploy_caasp4")
```

Comment:
I would prefer to keep `caasp4` everywhere and just call it in `deploy_caasp4` , too. That way the PR will be pretty simple.

Reply:
Well it would be less simple now to revert previous changes :-)

Is there a better reason to keep caasp4 everywhere?